datafusion/sql/src/utils.rs

```diff
@@ -268,6 +268,7 @@ pub(crate) fn value_to_string(value: &Value) -> Option<String> {
         Value::SingleQuotedString(s) => Some(s.to_string()),
         Value::DollarQuotedString(s) => Some(s.to_string()),
         Value::Number(_, _) | Value::Boolean(_) => Some(value.to_string()),
+        Value::UnicodeStringLiteral(s) => Some(s.to_string()),
```

Comment:
I don't really understand why `value_to_string` supports (returns `Some(String)`) for some types of string, but not others, so I'm not clear whether `Value::UnicodeStringLiteral` should return `Some` or `None`?

Reply:
I suspect this was not carefully designed and likely is not an intentional oversight

Reply:
I think what you have here is ok